### PR TITLE
Upgrade to Frida 15.0.0

### PR DESCRIPTION
**Pull request by oleavr**

### CODE REVIEW

**Comment by enovella** [comment]
LGTM. However, CI is complaining about the Windows build.

**Comment by radare** [comment]
The windows build fails because its using the last release of r2 instead of r2 from git. And its missing a new api. I can probably ifdef that to fix the build instead of enforcing newer r2

**Comment by trufae** [comment]
Let's burn the bridge!

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 include config.mk
 
 r2_version=$(VERSION)
-frida_version=14.2.18
+frida_version=15.0.0
```
Comment (trufae) [inline Makefile:4]:
```suggestion
frida_version=15.0.2
```

File: build.bat [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @echo off
 setlocal EnableDelayedExpansion
 set R2_BASE=""
-set frida_version=14.2.17
+set frida_version=15.0.0
```
Comment (trufae) [inline build.bat:4]:
```suggestion
set frida_version=15.0.2
```

**Review by trufae — APPROVED** [review]

**Pull request merged**